lib/chef/resource/chef_client_cron.rb

```diff
@@ -98,7 +98,7 @@ class ChefClientCron < Chef::Resource
       property :splay, [Integer, String],
         default: 300,
         coerce: proc { |x| Integer(x) },
-        callbacks: { "should be a positive number" => proc { |v| v > 0 } },
+        callbacks: { "should be a positive number" => proc { |v| v >= 0 } },
```

Comment:
Could you update the description here and in the other resources  to indicate that a 0 value will disable splay?

Reply:
description mentions settting time in seconds to wait so 0 would imply wait no seconds or disabled.